includes/class-newspack-popups-api.php

```diff
@@ -105,6 +102,45 @@ public function register_api_endpoints() {
 		);
 	}
 
+	/**
+	 * Recursively sanitize an array of arbitrary values.
+	 *
+	 * @param array $array Array to be sanitized.
+	 * @return array Sanitized array.
+	 */
+	public static function sanitize_array( $array ) {
+		foreach ( $array as $key => $value ) {
```

Comment:
This loop will not update the values from the given array. This can either change to `&$value` or store the sanitized values in `$array[ $key ]`.

Reply:
Whoops, good catch! I forgot a line. Fixed in 3a2fb9e.